scripts/get

```diff
@@ -94,8 +94,54 @@ download_and_install() {
     curl -sSfL "$TARBALL_URL" | tar xfz - --strip-components=1 -C "$TMPDIR"
 
     echo Installing CRI-O
-    make -C "$TMPDIR"
+    pushd "$TMPDIR"
+
+    PREFIX=/usr/local
+    ETCDIR=/etc
+    CONTAINERS_DIR=$ETCDIR/containers
+    CNIDIR=$ETCDIR/cni/net.d
+    BINDIR=$PREFIX/bin
+    MANDIR=$PREFIX/share/man
+    OCIDIR=$PREFIX/share/oci-umount/oci-umount.d
+    SELINUX=$(selinuxenabled 2>/dev/null && echo -Z)
+    BASHINSTALLDIR=$PREFIX/share/bash-completion/completions
+    FISHINSTALLDIR=$PREFIX/share/fish/completions
+    ZSHINSTALLDIR=$PREFIX/share/zsh/site-functions
+    SYSTEMDDIR=$PREFIX/lib/systemd/system
+    OPT_CNI_BIN_DIR=/opt/cni/bin
```

Comment:
is there any chance we can have a shared source file for these, in case the makefile values change?

Reply:
Hm, we could ship two additional scripts: `install` and `uninstall`. The `install` script could be re-used by the `get` script, but we would have to invoke an additional `curl` command... Not sure about how to solve it in a cleaner way...

Reply:
We can have something like `scripts/install-paths`:
```make
# NOTE this file is sourced from make and shell
PREFIX=/usr/local
ETCDIR=/etc
CONTAINERS_DIR=$ETCDIR/containers
....
```

and we can use this from Makefile:
```make
include ./scripts/install-paths

...

install: 
        install ${SELINUXOPT} -D -m 755 bin/crio $(DESTDIR)$(BINDIR)/crio
.....
```

as well as from the `scripts/get`:
```sh
. install-paths
```

That way we at least have a single source for all these paths.

NOTE that currently `DESTDIR` is not handled correctly by the Makefile. It should never be included into `PREFIX`, but rather specified directly as in my example above.

Reply:
> and uninstall

I would drop the `uninstall` target from the makefile. It is not very robust anyway, as no one keeps track of what's installed. For example, `make DESTDIR=$(pwd)/inst install` followed by `make PREFIX=/usr uninstall` might do some real harm.

Reply:
Removed the uninstall part from the makefile and changed the implementation to just provide an `install` script.